docs/regenerating-google-oauth-credentials.md

```diff
@@ -0,0 +1,20 @@
+# Regenerating Google OAuth credentials
+
+The `GOOGLE_OAUTH_CREDENTIALS` environment variable contains a JSON
+representation of the OAuth credentials used to connect to the email
+account to check for email alerts.
+
+If these credentials stop working (for example, they are revoked), they
+can be regenerated using the following steps:
+
+1. Run `bundle exec rake get_oauth_url` to get a Google authorisation
+   URL.
+2. Visit this URL in a browser while logged in as the account that
+   will be checked for email alerts.
```

Comment:
I wonder if you want to make this step 1 as it seems an easy one to get caught up on. Maybe even recommend doing it in a private browser session

Reply:
I've added a new step 1 to mention this.